src/perftest_parameters.c

```diff
@@ -544,6 +544,11 @@ static void usage(const char *argv0, VerbType verb, TestType tst, int connection
 		printf("      --tclass=<value> ");
 		printf(" Set the Traffic Class in GRH (if GRH is in use)\n");
 
+		if (connection_type != RawEth) {
+			printf("      --flow_label=<value> ");
+			printf(" Set the flow_label in GRH (if GRH is in use)\n");
```

Comment:
@HassanKhadour 
Thanks for review. 
Updated to be ```if GRH is in use```.

Reply:
Thanks!